sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -918,22 +970,45 @@ template <typename T, uint8_t NElts = 1,
           cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none>
 __ESIMD_API void lsc_block_store(T *p, __ESIMD_NS::simd<T, NElts> vals,
                                  __ESIMD_NS::simd_mask<1> pred = 1) {
-  detail::check_lsc_vector_size<NElts>();
   detail::check_lsc_data_size<T, DS>();
-  detail::check_lsc_cache_hint<detail::lsc_action::store, L1H, L3H>();
+  detail::check_lsc_cache_hint<detail::lsc_action::load, L1H, L3H>();
```

Comment:
Is this intentional replacement of lsc_action::store with lsc_action::load for this lsc_block_store() function?
It looks like an error or copy-paste error to me.